script/install_05_polyaxon_cert_manager.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+cert_manager_version="0.13.1"
+
+info() {
+    echo "$(tput bold)====> $1$(tput sgr0)"
+}
+
+info "Skip cert-manager installation."
+exit 0
```

Comment:
Again back and forth. cert-manager is required only for https which doesn't make much sense when we don't have a domain and access polyaxon only from inside the lab.

Reply:
I would like to allow access from outside as well. Working remotely. Would we now have to ssh into UoB servers first?